apps/client/src/sttr_client/models/checker.js

```diff
@@ -50,6 +50,33 @@ class Checker {
     return this._stack;
   }
 
+  /**
+   * Get data from the stack and return a list of questionIds with the given answers.
+   * @returns {({string: boolean|string|number|[string]})}  - a list of answers
+   */
+  getData() {
+    return this.stack.reduce((acc, question) => {
+      acc[question.id] = question.answer;
+      return acc;
+    }, {});
+  }
+
+  /**
+   * Set data from context to the current stack
+   * @returns {void} -
+   */
+  setData(answers) {
```

Comment:
Shall we rename this to `setQuestionAnswers()`? @svenjens @afjlambert 

This is too general and we're probably going to need more of these getters/setters in the near future